frontend/src/widgets/OverviewWidgetField.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Tooltip as TrussWorksToolTip } from '@trussworks/react-uswds';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faQuestionCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import './OverviewWidgetField.scss';
+
+import Tooltip from '../components/Tooltip';
+import colors from '../colors';
+
+export function OverviewWidgetField({
```

Comment:
There's a duplicative component in frontend/src/widgets/DashboardOverview.js, can we condense

Reply:
Will leave as is for now as this will need to be updated to use the new overview widget in the future.

Reply:
can you explain what you mean?